src/rewrite.rs

```diff
@@ -1,6 +1,7 @@
 // A generic trait to abstract the rewriting of an element (of the AST).
 
 use std::cell::{Cell, RefCell};
+use std::fmt::{self};
```

Comment:
Is there a specific reason we're importing this as `std::fmt::{self};` and not `std::fmt;`? We can probably just `impl std::fmt::Display for MacroErrorKind` below and avoid the `use` altogether.

Reply:
I agree that it’s better to avoid the additional use here. Thank you for pointing it out. I updated the code.